spec/models/manageiq/providers/ibm_cloud/vpc/network_manager/cloud_network_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+describe ManageIQ::Providers::IbmCloud::VPC::NetworkManager::CloudNetwork do
+  let(:ems) do
+    FactoryBot.create(:ems_ibm_cloud_vpc, :provider_region => "us-east").tap do |ems|
+      ems.authentications << FactoryBot.create(:authentication, :auth_key => 'IBM_CLOUD_VPC_API_KEY')
+    end
+  end
+
+  let(:cloud_network) do
+    FactoryBot.create(:cloud_network_ibm_cloud_vpc,
+                      :ext_management_system => ems.network_manager,
+                      :name                  => 'test',
+                      :ems_ref               => 'test_id')
```

Comment:
Usually anything like this that is required is auto-generated by the factory so you only need to pass something in if you care about the value.

Basically you can drop the `:name` and `:ems_ref` unless you need the to actually equal `test` and `test_id` for the rest of your specs